index.html

```diff
@@ -3,53 +3,188 @@
   <head>
     <title>Girma's Page</title>
     <meta charset="UTF-8" />
-    <meta name="keywords" content="HTML, CSS, JavaScript" /> 
-    <link rel="stylesheet" type="text/css" href="css/index.css"> 
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto">
+    <link rel="stylesheet" type="text/css" href="css/index.css" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Roboto"
+    />
   </head>
   <body>
     <nav>
       <ul>
         <li><a href="#About">About</a></li>
         <li><a href="#Experience">Experience</a></li>
```

Comment:
The navigation looks good. I did notice that the experience link isn't quite working. I think it's because the experience section is being defined by the class attribute and not the id. I'd try adding an id and seeing if that fixes it.